fuzzing/scripts/custom-build.sh

```diff
@@ -222,6 +219,40 @@ printf "|_|  |_| \_\____)____) |_| /_/   |_|   |____)\n\n"
 printf "${ansi_reset}"
 printf "               Custom Build\n"
 
+clang_version_regex='.*clang version ([0-9]+)\.[0-9]+.*'
+
+if [[ "${CC}" =~ .*"clang".* ]]; then
+    cc="${CC}"
+else
+    cc="clang"
+fi
+print_info "cc" "${cc}"
+cc_version_output=$("${cc}" --version)
+if [[ "${cc_version_output}" =~ ${clang_version_regex} ]]; then
+    cc_version="${BASH_REMATCH[1]}"
+    print_info "cc version" "${cc_version}"
+else
+    print_info "cc version not detected"
+fi
+
+if [[ "${CXX}" =~ .*"clang".* ]]; then
+    cxx="${CXX}"
+else
+    cxx="clang++"
+fi
+print_info "cxx" "${cxx}"
+cxx_version_output=$("${cxx}" --version)
+if [[ "${cxx_version_output}" =~ ${clang_version_regex} ]]; then
+    cxx_version="${BASH_REMATCH[1]}"
+    print_info "cxx version" "${cxx_version}"
+    if [ "${cxx_version}" -lt "10" ]; then
+        print_info "cxx version too old. Need clang 10 or newer."
+        exit 66
```

Comment:
Is 66 a common return value for such situations?  Otherwise please add a comment to mark this as an ad-hoc value.

Reply:
I mostly did this because the other uses of `exit` in this script already used 66. I think this is arbitrary, some other tool may have been looking for this value, but this is a user script not intended for automation, so it doesn't really matter what the return code is. I think previously 64-113 were "user defined" in the sense that nothing else really used them, but now `/usr/include/sysexits.h` generally defines meanings for 64-78, though of course there's nothing really all the special about these particular meanings. I'm just gong to leave this as 66 for the moment for consistency.